ethereum/core/src/main/java/org/hyperledger/besu/ethereum/trie/diffbased/common/DiffBasedAccount.java

```diff
@@ -60,6 +60,25 @@ public DiffBasedAccount(
     this.immutable = !mutable;
   }
 
+  public DiffBasedAccount(
```

Comment:
according to intellij, the 3rd and 4th constructors for DiffBasedAccount are unused.  Can we drop those and add javadoc to the remaining constructors that indicate in what cases we want to use one over the other

Reply:
updated